src/main/java/com/basilisk/frontend/views/LoginView.java

```diff
@@ -0,0 +1,89 @@
+package com.basilisk.frontend.views;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.PasswordField;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.router.Route;
+
+@Route("")
```

Comment:
Remove the (""), You just need @Route if its the original page

Reply:
@Route does work since the page does not land on the LoginView. Instead, use @Route("").

Reply:
Okay as we discussed, we will just leave it as @Route(""). Will approve